pkg/didcomm/protocol/introduce/states.go

```diff
@@ -70,8 +75,12 @@ func (s *start) CanTransitionTo(next state) bool {
 	return next.Name() == stateNameArranging || next.Name() == stateNameDelivering || next.Name() == stateNameDeciding
 }
 
-func (s *start) Execute(msg *service.DIDCommMsg) (state, error) {
-	return nil, errors.New("start execute: not implemented yet")
+func (s *start) ExecuteInbound(msg *service.DIDCommMsg) (state, error) {
+	return nil, errors.New("start ExecuteInbound: not implemented yet")
+}
+
+func (s *start) ExecuteOutbound(_ *service.DIDCommMsg) (state, error) {
```

Comment:
@rolsonquadras @llorllale @soluchok Are we also planning to split Execute in the DID Exchange protocol implementation? Where does the outbound destination become needed?

Reply:
yeah, there is an issue https://github.com/hyperledger/aries-framework-go/issues/501

Reply:
Execute Outbound should take `service.Destination` arg as well.